docs/examples/warp_gpu.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
If you want to make it a part of our example set please add more description why certain things are done.